tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -114,9 +113,20 @@ object QualOutputWriter {
   val COMPLEX_TYPES_STR = "Complex Types"
   val NESTED_TYPES_STR = "Nested Complex Types"
   val READ_SCHEMA_STR = "Read Schema"
+  val NONSQL_DUR_STR = "NONSQL Task Duration Plug Overhead"
```

Comment:
It looks like "Plug Overhead" is extra here.

Reply:
oh sorry, this is supposed to be Plus Overhead

Reply:
fixed the typo